gcexport.py

```diff
@@ -765,12 +832,12 @@ def logging_verbosity(verbosity):
     for handler in logger.handlers:
         if isinstance(handler, logging.FileHandler):
             # this is the logfile handler
-            level = logging.DEBUG if verbosity > 0 else logging.INFO
+            level = logging.DEBUG if verbosity and verbosity > 0 else logging.INFO
```

Comment:
Why this change?

Reply:
This change was a consequence from running `python2 -3` before the first tries with python3, I didn't really try if it's needed with python3

Reply:
Could you please paste output of the running `python2 -3` without this change? I truly looks weird to me.

Reply:
Seems if `verbosity` isn't set on the command line it's `None` and can't be compared to 0:

    gcexport.py:835: DeprecationWarning: comparing unequal types not supported in 3.x
      level = logging.DEBUG if verbosity > 0 else logging.INFO

Reply:
we could change one line from:
`parser.add_argument('-v', '--verbosity', action='count', help='increase output verbosity')`
to:
`parser.add_argument('-v', '--verbosity', action='count', default=0, help='increase output verbosity')`

now we can remove the extra check(s):
from:
`level = logging.DEBUG if verbosity and verbosity > 0 else logging.INFO`
to
`level = logging.DEBUG if verbosity > 0 else logging.INFO`

Reply:
thx @telemaxx, that simplifies the code a lot